src/languages/gcode.js

```diff
@@ -51,13 +51,7 @@ export default function(hljs) {
     },
     {
       className: 'symbol',
-      variants: [
-        {
-          begin: 'N',
-          end: '\\d+',
-          illegal: '\\W'
-        }
-      ]
+      begin: 'N\\d+'
```

Comment:
```suggestion
      match: /N\d+/
```

Reply:
Should I change the other 10 regexes with single quotes in that file, too?

Reply:
Update: I found heaps more issues with samples of gcode I could find online. I will submit a PR with substantial changes. Until this, this suggestion was implemented and this PR can stay as small as it is.